products/keda.md

```diff
@@ -0,0 +1,70 @@
+---
+title: KEDA
+category: server-app
+tags: kubernetes
+permalink: /keda
+releasePolicyLink: https://keda.sh/docs/latest/operate/cluster/
+changelogTemplate: "https://github.com/kedacore/keda/releases/tag/v__LATEST__"
+eolColumn: false
+releaseColumn: Latest
+releaseDateColumn: Released
+
+auto:
+  - git: https://github.com/kedacore/keda.git
+
+identifiers:
+  - purl: pkg:github/kedacore/keda
+  - purl: pkg:github/kedacore/charts
```

Comment:
Lets add the OCI images as well. (Based this on the [official release manifest](https://github.com/kedacore/keda/releases/download/v2.11.2/keda-2.11.2.yaml))
```suggestion
  - purl: pkg:github/kedacore/charts
  - purl: pkg:oci/keda-admission-webhooks?repository_url=ghcr.io/kedacore
  - purl: pkg:oci/keda-metrics-apiserver?repository_url=ghcr.io/kedacore
  - purl: pkg:oci/keda?repository_url=ghcr.io/kedacore
```

Reply:
Agree! Thanks.